.github/workflows/release-workflow.yml

```diff
@@ -0,0 +1,120 @@
+name: Release workflow
+# This workflow is triggered on creating tags to master or a opendistro release branch
+on:
+  #  push:
+  #    tags:
+  #      - 'v*'
```

Comment:
remove?

Reply:
removed on push event and uncommented tag event trigger now that action is tested.